plugins/woocommerce-blocks/tests/e2e/tests/templates/checkout-template.block_theme.spec.ts

```diff
@@ -21,7 +21,7 @@ test.describe( 'Test the checkout template', () => {
 		await expect(
 			page
 				.frameLocator( 'iframe[title="Editor canvas"i]' )
-				.locator( 'h1:has-text("Checkout")' )
+				.locator( 'div:has-text("Place Order · <price/>")' )
```

Comment:
I've found it a good practice to let Playwright determine the selectors for me as it takes care of the recommended practices and the selector uniqueness on the page. It sometimes needs to be tweaked a little bit (e.g., we use `editor.canvas.*` instead of explicit `page.frameLocator( 'iframe[title="Editor canvas"i]' ).*`), but it does the job right most of the time.

I've recorded a little vid to demo how I use the Plawright codegen for that in the WC context, as it's easier than writing the steps! :D 

https://github.com/woocommerce/woocommerce/assets/1451471/95452021-b586-49df-b15e-49d72714e0b7

Reply:
Thanks a lot for pointing this out, @WunderBart. I went ahead and replaced all `page.frameLocator( 'iframe[title="Editor canvas"i]' ).*)` sections.